modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CSharpRefactorClientCodegen.java

```diff
@@ -208,6 +210,26 @@ public CSharpRefactorClientCodegen() {
 
     @Override
     public void processOpts() {
+        // mapped non-nullable type without ?
```

Comment:
I think this block has to go into the constructor. Otherwise, the user won't be able to override types for things like DateTime, array, list, and map.